src/modules/displayReserve.js

```diff
@@ -0,0 +1,39 @@
+import countReserve from './countReserve.js';
+
+const displayReserve = async () => {
+  const reserveId = document.querySelector('.movie-title').id;
+  const url = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/IR7MovTrVQtBQVyC6UTK/reservations?item_id=${reserveId}`;
+
+  const reservationList = document.querySelector('.movie-reservation');
+  reservationList.innerHTML = '';
+
+  try {
+    const response = await fetch(url, {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+
+    const reservationData = await response.json();
+
+    reservationData.forEach((reserve) => {
+      reservationList.innerHTML += `
+        <li class="reservation-list">
+          <p class="date">${reserve.date_start}</p>
+          <p>-</p>
+          <p class="date">${reserve.date_end}</p>
+          <p>by:</p>
+          <p class="user-name">${reserve.username}</p>
+
+
+        </li>`;
+    });
```

Comment:
- [ ] [OPTIONAL] Just another observation I have made and believe could help improve the aesthetics and user experience of your solution, positioning all elements at the center would be more accurate just every other is except the image. This is applicable to your Comment Pop up page too. Can you position the image at the center of the available viewport/width?

![image](https://github.com/NoelLincoln/movie-safari/assets/99715640/bd7b4204-0b65-426a-ae33-a6e695597ee6)